src/main/java/UStore/GiftCard.java

```diff
@@ -0,0 +1,43 @@
+package UStore;
+
+public class GiftCard implements RewardsItem{
```

Comment:
Should we also add a method so that the value on this gift card could somehow be transfered to this customer's account so that the user could actually use it?

Reply:
But this is a ticket reservation system instead of the Starbucks or T card system, I think if you want to have something that can be used as money for customer's purchase, we could consider adding new rewards item like a prepaid card that is specified for this system.